.travis.yml

```diff
@@ -1,41 +1,98 @@
-language: python
-matrix:
-  include:
-  - os: linux
-    python: 3.6
-    dist: bionic
-  - os: linux
-    python: 3.7
```

Comment:
We don't test for python 3.8? Not urgent now, just asking in general

Reply:
I didn't generally test py38 because not everything supports it yet, but everything here should. I'll add it.

Reply:
@vigji I'm having trouble installing on py38 (on ubuntu), the issue seems to be with statsmodels/numpy (even though both have py38 wheels). Have you been using this on python 3.8?

Reply:
[Doesn't seem to be Ubuntu specific](https://travis-ci.com/github/brainglobe/brainatlas-api/builds/168447654), so I'm dropping this for now. 

I think it'll be fine when the allensdk dependency is removed.